autogpts/forge/forge/__main__.py

```diff
@@ -36,9 +35,8 @@
 if __name__ == "__main__":
     print(logo)
     port = os.getenv("PORT", 8000)
-    LOG.info(f"Agent server starting on http://localhost:{port}")
+    logger.info(f"Agent server starting on http://localhost:{port}")
```

Comment:
this won't do anything unless the logger is configured, e.g. using `configure_logging`
```suggestion
    configure_logging()
    logger.info(f"Agent server starting on http://localhost:{port}")
```